docs/README.md

```diff
@@ -18,18 +18,34 @@ This app integrates the [Adyen for Platforms](https://docs.adyen.com/platforms)
 
 ⚠️ Before you can use the Adyen Platforms app, the [Adyen Payment connector](https://github.com/vtex-apps/connector-adyen) must be installed and configured in you VTEX account.
 
+### Contacting Adyen
+
+Users must first contact Adyen to get their Adyen for Platforms set up. You will know when Adyen for Platforms is setup when a merchant account has the Platform tab on the sidebar. [Adyen for Platforms Quick Start Guide](https://docs.adyen.com/platforms/quick-start)
+
 ### Installing the App
 
 1. Install this app in the desired account using the CLI command `vtex install vtex.adyen-platforms`.
 2. In your admin sidebar, select `Adyen for Platforms` under the **Marketplace** section.
```

Comment:
Let's recommend using the admin search function here

Reply:
This platform sidebar tab is in Adyen's own admin panel, not our app panel

Reply:
But this step is talking about navigating to the VTEX Adyen admin panel to enter the app settings